ee/agent/startupsettings/writer.go

```diff
@@ -61,6 +61,7 @@ func (s *startupSettingsWriter) setFlags() error {
 	for flag, getter := range s.storedFlags {
 		updatedFlags[flag.String()] = getter()
 	}
+	updatedFlags["use_tuf_autoupdater"] = "enabled" // Hardcode for backwards compatibility
```

Comment:
Might be worth a comment about what version this is for. I can imagine wanting to drop it in the future.